content/en/posts/outer-misalignment.md

```diff
@@ -0,0 +1,11 @@
+Intermediate
```

Comment:
This is "Basic" not intermediate, which is good! IMHO "Basic" is the most important section. I really like what you wrote because it strikes the tone that I am aiming for myself :slightly_smiling_face: .